### PR TITLE
[Round 4 Method References Static]Lps 183180

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-183180

### CODE REVIEW

File: modules/apps/source-formatter-suppressions.xml [hunk 1]
```diff
@@ -26,7 +26,6 @@
 		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtil\.java" />
 		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/java/com/liferay/dynamic/data/mapping/taglib/internal/servlet/ServletContextUtil\.java" />
 		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/java/com/liferay/dynamic/data/mapping/taglib/internal/util/PortletDisplayTemplateUtil\.java" />
-		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/util/PortletDisplayTemplateUtil\.java" />
```
Comment (dantewang) [inline modules/apps/source-formatter-suppressions.xml:29]: @slnn Is the one above this also modified in this PR? Could you remove it from this file also and try SF? I think we should remove it as long as we convert the util to non-component.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/source-formatter-suppressions.xml (hunk 1 above)
Comment (slnn) [reply modules/apps/source-formatter-suppressions.xml:29]: Done

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:forms

**Comment by zhangyan0701** [comment]
ci:test:site-template

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by tinatian** [comment]
@slnn @dantewang Please add this to your task list : There are more than one PortletDisplayTemplateUtil existed in different modules, please take a deep look into it and see if we can centralize the uages.

Thanks.
Tina.

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/134308

**Comment by slnn** [comment]
> @slnn @dantewang Please add this to your task list : There are more than one PortletDisplayTemplateUtil existed in different modules, please take a deep look into it and see if we can centralize the uages.
> 
> Thanks. Tina.

OK, got it

**Comment by dantewang** [comment]
@slnn These PortletDisplayTemplateUtil were created by our team in the history. Please double check if the reason we made these changes still exists. If not, just centralize it.

cc @tinatian

**Pull request closed without merging**